app/main.py

```diff
@@ -2,0 +1,6 @@
+def copy_file(command: str) -> None:
+    files = command.split(" ")
```

Comment:
Ensure that the `.split()` method is called only once on the input string, as per the checklist's recommendation.